doc/contribute/index.rst

```diff
@@ -761,3 +761,66 @@ Code component README template
 ==============================
 
 .. literalinclude:: code_component_README
+
+
+Contribution Roles and Responsibilities
+***************************************
+
+Zephyr community has a number of working groups and community members that
+work together in a development process flow to reviews PRs and Issue reports
+(Features/Enhancements/bugs). These PRs and Issue reports are used to help
+with the feature enhancements and quality improvements of Zephyr through
+regular releases as outlined in the Program Management wiki.
+
+To manage the volume of PRs and Issue reports within the project there are
+specific roles and responsibilities that are followed within the development
+process flow.
+
+PRs
+===
+
+**Zephyr Community**
+    • Provide timely review and disposition of submitted PRs that are not
+      marked as <DNM>.
+
+**PR Contributor**
+    • Provide timely response to questions and engagement on the PR.
+    • Work with maintainers and reviewers to resolve any issues.
+
+Bug Reports
+===========
+
+**Zephyr Community**
+    • Provide timely review of the bug report. Engage with submitter to gather
+      sufficient information on the problem to be able to reproduce the bug
+      and to assign a relative priority to the bug.
+    • Addressing the bug will be handling in the normal development flow and
+      release management
+
+**Bug Reporter**
+    • Provide timely responses to questions on the bug report from the
+      community.
+
+Feature/Enhancement Requests
+============================
+
+**Zephyr Community**
+    • Provide timely review and disposition of submitted feature requests.
+
+**Feature/Enhancement Requestor**
+    • Provide timely response to questions and requests for information on
+      feature/enhancement request.
+
+Contributor Timely Responses
```

Comment:
There lots of documentation regarding this in doc/development_process/dev_env_and_tools.rst

I suggest to either combine or make sure we do not duplicate.

Reply:
Okay. I'll review that section as well.